i18n/config.json

```diff
@@ -0,0 +1,370 @@
+[
+  {
+    "code": "en",
+    "hrefLang": "en",
+    "name": "English",
+    "localName": "English",
+    "langDir": "ltr",
+    "dateFormat": "MM/DD/YYYY"
+  },
+  {
+    "code": "ar",
+    "hrefLang": "ar",
+    "name": "Arabic",
+    "localName": "العربية",
+    "langDir": "rtl",
+    "dateFormat": "MM/DD/YYYY"
```

Comment:
Shouldn't date formatting vary by language?

Reply:
Not sure if we need to rely on these? We have the `getLocaleTimestamp` helper for displaying date formats.

Reply:
Yes, I've added these fields because they were required by the `gatsby-theme-i18n` plugin. For now, we are not using `langDir` and `dateFormat` from here. Those things are still working as before.

- I agree we should not use `dateFormat` from this file. We should keep using what we currently have.
- However, I think we could consume `langDir` from this config file instead of what we are currently doing (function `isLangRightToLeft`).

Reply:
Gotcha - I'm on board.